aws/resource_aws_lb_test.go

```diff
@@ -194,6 +194,37 @@ func TestAccAWSLB_LoadBalancerType_Gateway(t *testing.T) {
 	})
 }
 
+func TestAccAWSLB_IPv6SubnetMapping(t *testing.T) {
+	var conf elbv2.LoadBalancer
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_lb.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t); testAccPreCheckElbv2GatewayLoadBalancer(t) },
```

Comment:
i think this last precheck can be safely removed since this tests a network LB but i'll double check 👍

Reply:
OK yep ran the tests in `us-gov-west-1` in case and the tests pass w/o the gateway precheck

Reply:
@bill-rich just one last note to update this to
```suggestion
		PreCheck:          func() { testAccPreCheck(t) },
```
since we can run these in alternate regions/gov cloud afict. unless you've experienced something different 😃